src/main/java/com/exasol/adapter/document/documentfetcher/files/JsonDocumentFetcher.java

```diff
@@ -16,7 +17,7 @@
  * {@link DocumentFetcher} for JSON files.
  */
 public class JsonDocumentFetcher extends AbstractFilesDocumentFetcher<JsonNodeVisitor> {
-    private static final long serialVersionUID = 9127449898816112421L;
+    private static final long serialVersionUID = 9127449898816112421L;//
```

Comment:
Extra slashes (`//`) at the end of the line.

Reply:
They are needed to make a change so that the serialVersionUID checker does not report an error.